components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Engine/LegacyPublisherConfigServicesTests.cs

```diff
@@ -267,5 +270,139 @@ public class LegacyPublisherConfigServicesTests {
                 .Should()
                 .Be(2);
         }
+
+        [Fact]
+        public async Task PublishNodesStressTest() {
+            var legacyCliModelProviderMock = new Mock<ILegacyCliModelProvider>();
+            var agentConfigProviderMock = new Mock<IAgentConfigProvider>();
+            var identityMock = new Mock<IIdentity>();
+            var newtonSoftJsonSerializer = new NewtonSoftJsonSerializer();
+            var jobSerializer = new PublisherJobSerializer(newtonSoftJsonSerializer);
+            var publishedNodesJobConverter = new PublishedNodesJobConverter(TraceLogger.Create(), newtonSoftJsonSerializer);
+
+            string tempPublishedNodesFile = Path.GetTempFileName();
```

Comment:
we should probably delete this file as soon as the orchestrator is created and makes use of it. if we delete it only in the end of the testcase we might encounter exceptions and end-up with a bunch of undeleted empty files. 
Alternatively put the entire code in a try...finally block for cleanup.

Reply:
Added removal of temporary file on test cleanup.